exam/report/batch_result_report.py

```diff
@@ -34,7 +35,7 @@ def pass_student(self, year, standard_id):
                      'fail_student': len(exam_result_fail.ids) or 0.0}]
 
     @api.model
-    def get_report_values(self, docids, data=None):
+    def _get_report_values(self, docids, data=None):
```

Comment:
can you add dcostring here because it becomes hard to understand the reason behind change.